concordium_p2p_rpc.proto

```diff
@@ -80,7 +80,6 @@ message NodeInfoResponse {
   IsInBakingCommittee consensus_baker_committee = 7;
   bool consensus_finalizer_committee = 8;
   google.protobuf.UInt64Value consensus_baker_id = 9;
-  google.protobuf.StringValue staging_net_username = 50;
```

Comment:
I don't think we should change the API. We should mark the field as deprecated.

Reply:
https://developers.google.com/protocol-buffers/docs/proto3

Reply:
Or alternatively make it reserved https://developers.google.com/protocol-buffers/docs/proto3#reserved 

But I think the most painless transition will be to mark it deprecated.

Reply:
I suppose you're right if it's "already being used". I'll mark the field as deprecated.